README.md

```diff
@@ -118,6 +118,10 @@ Check out our examples:
 
 Looking for a specific example? [Request one here!](https://github.com/Mirrorful/mirrorful/issues/357)
 
+### ESLint Plugin
+
+We have an ESLint plugin with a recommended configuration that's ready to integrate with any project and UI library. It will detect hard-coded color strings throughout your project and help you convert them to the Mirrorful theme with a simple `npm run lint`. Learn more [here](/packages/eslint-plugin/).
+
```

Comment:
```suggestion
### Mirrorful ESLint Plugin
We have an ESLint plugin with a recommended configuration that's ready to integrate with any project and UI library. It will detect hard-coded color strings throughout your project and help you convert them to the Mirrorful theme with a simple `npm run lint`. [Learn more](/packages/eslint-plugin/).
```

Also on line 58, can you delete - `🔜 **Eslint rules**` and instead add a bullet and a hyperlink to the package.